vagrant_support/cron.yml

```diff
@@ -0,0 +1,45 @@
+- name: set cron mailto
+  cron:
+    env: yes
+    name: MAILTO
+    value: vagrant@localhost
```

Comment:
A email sending can be configured on vagrant IIRC. But I'm not sure, if this is configured for it. @dylanwh?